tools/@aws-cdk/enum-updater/lib/static-enum-mapping.json

```
@@ -157,7 +157,7 @@
     "VpcEndpointIpAddressType": {
       "cdk_path": "aws-cdk/packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts",
       "sdk_service": "ec2",
-      "sdk_enum_name": "IpAddressType",
+      "sdk_enum_name": "DnsRecordIpType",
```

Removing this as it is an incorrect mapping. Will add the new manual mapping in a following change.
```suggestion
      "sdk_enum_name": "IpAddressType",
```